resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/PVTestsSuite.scala

```diff
@@ -71,7 +71,7 @@ private[spark] trait PVTestsSuite { k8sSuite: KubernetesSuite =>
                 .withMatchExpressions(new NodeSelectorRequirementBuilder()
                   .withKey("kubernetes.io/hostname")
                   .withOperator("In")
-                  .withValues("minikube", "m01", "docker-for-desktop", "docker-desktop")
+                  .withValues("minikube", "m01", "docker-desktop")
```

Comment:
Note: `docker-for-desktop` was already [renamed to](https://github.com/docker/for-mac/issues/4089) `docker-desktop`  since docker desktop 2.4 for win and mac, so we can drop `docker-for-desktop` in here safely.

Reply:
Please don't remove this.